src/data/showcases.js

```diff
@@ -1127,6 +1127,16 @@ export const Showcases = [
     source: null,
     tags: ["lending", "token", "marketplace", "nftsupport"],
   },
+  {
+    title: "NFTify",
+    description:
+      "The leading no-code NFT marketplace solution aims to empower all Web3 businesses to launch and trade their NFT assets without friction, coding skills – at the lowest possible costs.",
+    preview: require("./showcase/nftify.png"),
+    website: "https://nftify.network/",
+    source: null,
+    tags: ["marketplace", "minting", "nftproject"],
+    developmentCompany: "Ekotek",
```

Comment:
```suggestion
```
Here is the error you need to fix: from this arbitrary, non-existent key.  I'll wait for you to commit this & ensure that it builds.
```
[ERROR] Docusaurus server-side rendering could not render static page with path /showcase/.
[ERROR] Docusaurus server-side rendering could not render static page with path /tools/.

Error: Showcase site with title=NFTify contains errors:
Site contains unknown attribute names=[developmentCompany]
```

Reply:
@rphair Sorry my boss insist on having the company name attached with this :sweat_smile: . I have removed it, please check. Thank you!